chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -73,6 +73,25 @@
 
   include_recipe "crowbar::prepare-upgrade-scripts"
 
+when "none"
+
+  Chef::Log.debug("Deleting the scripts created for the upgrade...")
+  [
+    "prepare-repositories",
+    "upgrade-os",
+    "shutdown-services-before-upgrade",
+    "delete-cinder-services-before-upgrade",
+    "evacuate-host",
+    "pre-upgrade",
+    "delete-pacemaker-resources",
+    "router-migration",
+    "post-upgrade",
+    "chef-upgraded"
+  ].each do |f|
+    file "/usr/sbin/crowbar-#{f}.sh" do
+      action :delete
+    end
+  end
```

Comment:
Yeah, somehow I think we should do it from the rails app. BTW, do we have somehow remove the "crowbar-upgrade.rb " recipe from the run_list(s) once all nodes are upgraded? (I remember that was somehow tricky for the last release)